### ISSUE TITLE
The documentation fails to build on ReadTheDocs

### ISSUE THREAD

**Issue by jtpio** [jupyter-widgets/ipywidgets#3667]
## Description

The documentation fails to build on ReadTheDocs.

## Reproduce

The docs seem to be failing to build recently. This was noticed in https://github.com/jupyter-widgets/ipywidgets/pull/3661#issuecomment-1371032922: https://readthedocs.org/projects/ipywidgets/builds/19070899/

The error message is:

```
Command killed due to timeout or excessive memory consumption
```

This also seems to be the case on `latest` and most of the PRs.

## Expected behavior

Docs should build on ReadTheDocs.

## Context

This was noticed in https://github.com/jupyter-widgets/ipywidgets/pull/3661#issuecomment-1371032922.

- ipywidgets version: latest
- Operating System and version: N/A
- Browser and version: Any

**Comment by bollwyvl** [comment]
See also #3656... it's probably worth updating a number of things towards making the docs more useful to both end users _and_ widget developers.

**Comment by bollwyvl** [comment]
Probably closed by #3673

**Comment by jtpio** [comment]
Nice, thanks!

Closing as fixed.
